src/index.html

```diff
@@ -1,13 +1,407 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" сontent="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#menu" class="icon icon--menu"></a>
+            <a href="/" class="logo">
+              <img
+                class="header__top logo__images"
+                src="images/logo.png"
+                alt="Miami Condo Kings"
+              >
+            </a>
```

Comment:
Add spaces between elements where needed
```suggestion
            <a href="#menu" class="icon icon--menu"></a>
            
            <a href="/" class="logo">
              <img
                class="header__top logo__images"
                src="images/logo.png"
                alt="Miami Condo Kings"
              >
            </a>
```